docs/scripts/generate_ts_examples/scripts/generate_ts_docs.ts

```diff
@@ -124,10 +124,18 @@ async function main() {
             file.replace('.ts', '.md')
         );
         
-        // Convert the file from .ts to .md
-        const blocks = await processTypeScriptFile(inputPath);
-        const markdown = await generateMarkdown(blocks);
-        await fs.writeFile(outputPath, markdown);
+        // Skip if output file already exists
+        try {
+            await fs.access(outputPath);
```

Comment:
this is not how gen folders work - if we update the source, it should update the gen target

Reply:
Gotcha will fix. Docs team made changes to the markdown directly for certain generations so I wanted to prevent those from being overwritten but will make those fixes made by them directly to the code files